commons/src/main/java/org/infinispan/commons/event/KVPEvent.java

```diff
@@ -0,0 +1,30 @@
+package org.infinispan.commons.event;
+
+import java.io.Serializable;
+
+public class KVPEvent<K, V> implements Serializable {
```

Comment:
It's modeled after the CustomEvent from the user guide but I guess I can change the name and perhaps even drop event from the name and package as suggested. Would it fit better in org.infinispan.commons(.util) this way?

I've added it to commons since it needs to be available from infinispan-jcache-remote/infinispan-hotrod-client and infinispan-server-hotrod and -commons is I think the only shared dep of the three. Adding a new maven module just for it didn't seem like a good option to me.